sklearn/datasets/_california_housing.py

```diff
@@ -90,7 +89,7 @@ def fetch_california_housing(
 
     Returns
     -------
-    dataset : :class:`~sklearn.utils.Bunch`
+    dataset : Class:`~sklearn.utils.Bunch`
```

Comment:
```suggestion
    dataset : :class:`~sklearn.utils.Bunch`
```
This is the notation for a [reference in sphinx](https://www.sphinx-doc.org/en/master/usage/restructuredtext/roles.html#role-ref) and should not be modified.